frontend/packages/dev-console/console-extensions.json

```diff
@@ -873,17 +873,6 @@
       "insertAfter": "topology-tab-section-builds-overview"
     }
   },
-  {
-    "type": "console.topology/details/tab-section",
-    "properties": {
-      "id": "topology-tab-section-vpas",
-      "tab": "topology-side-bar-tab-resource",
-      "provider": {
-        "$codeRef": "topology.useVpaTabSectionForTopologySideBar"
```

Comment:
This `useVpaTabSectionForTopologySideBar` export is broken, but I see that we have `packages/topology/src/components/workload/vpa-tab-section.tsx` which provides `useVpaSideBarTabSection` using the same signature `DetailsTabSectionExtensionHook`.

Looking at `packages/topology/console-extensions.json`
```json
  {
    "type": "console.topology/details/tab-section",
    "properties": {
      "id": "topology-tab-section-vpa-overview",
      "tab": "topology-side-bar-tab-resource",
      "provider": { "$codeRef": "workload.useVpaSideBarTabSection" }
    }
  },
```

Reply:
@logonoff I've updated my comment, I think that extension got moved over to topology package so it should be safe to remove.